app/assets/javascripts/discourse/app/lib/user-presence.js

```diff
@@ -25,8 +25,30 @@ export default function (maxUnseenTime) {
   }
 }
 
+const callbacks = [];
```

Comment:
Should we clean up this constant in our Qunit test suite? State leaks like this has been a constant source of pain in our QUnit test suite. While we're only adding a callback once in an initializer now, it may not be true in the future.

Reply:
there is a full bypass on all this code in test mode per: https://github.com/discourse/discourse/blob/45073891f33aacaba9bf97a14e782772538ec816/app/assets/javascripts/discourse/app/initializers/message-bus.js#L38-L41

Reply:
Ahh icic :+1: